Cargo.toml

```diff
@@ -36,7 +36,7 @@ required-features = ["scarb"]
 default = ["build-cli", "with-runtime"]
 build-cli = [
   "dep:clap",
-  "dep:tracing-subscriber",
+
```

Comment:
The dependency of `tracing-subscriber` should only be present for binaries, not the library. Configuring tracing is the responsibility of the binary, not the library.

Reply:
Is there a way to specify that a dependency is only used in a particular binary?

Reply:
Aah, with this: https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies, right? So i should just re add dep:tracing-subscriber to the build-cli feature

Reply:
I don't remember changing that, maybe it was changed automatically by `cargo add`? Anyway, I already pushed the fix. Thanks